bin/zeppelin-daemon.sh

```diff
@@ -94,7 +94,8 @@ function initialize_default_directories() {
     $(mkdir -p "${ZEPPELIN_PID_DIR}")
   fi
 
-  echo "Browse localhost:8080 in your browser"
+  IP_ADDR="$(ifconfig en0 | grep inet | grep -v inet6 | cut -d ' ' -f2)"
+  echo -e "Browse $IP_ADDR:8080 in your browser.\nIf you are testing on your local computer, use localhost:8080"
```

Comment:
@djKooks As you can see in [Apache Zeppelin Configuration](http://zeppelin.apache.org/docs/0.7.0-SNAPSHOT/install/install.html#apache-zeppelin-configuration), user can also change the server port using `ZEPPELIN_PORT ` and `zeppelin.server.port` value.  Can we update the port as well? Sorry about this. I should've told about it with my last comment.